### ISSUE TITLE
Luis Staging behaving as production and now I cant train not even in the portal

### ISSUE THREAD

**Issue by martinlarosa** [microsoft/botbuilder-tools#1401]
## Tool
Name: Dispatch
Version: 1.5.6
OS: MacOS 10.15.2

## Describe the bug
Yesterday I was trying to refresh dispatch, with version 1.5.2 I believe, to staging and then I was getting different results than from the portal. I guess that they were dirty with production data? Just a guess. 
Then just to circumvent that issue, today I updated the dispatch tool, created a new resource for authoring, imported the apps in there. Ran the dispatch tool. Now everything fails, dispatch tool fails when running were the output reads: 

> Training my-dispatch model...
> 
> ERROR
> Training failed

When I go to the portal I can delete intents but not train, and if I look into the Chrome console I see a lot of errors mostly 429.

Similar to this: https://stackoverflow.com/questions/57482652/luis-app-fails-to-train-app-training-failed-l-general-retrain
[bug]

**Comment by jwiley84** [comment]
Good morning, Martin!

I am unable to repro the LUIS issue (I noticed the Stack Overflow issues with it as well). Before we dig down into the dispatch problem, can you test training your LUIS model for me this morning, and let me know if it is still unavailable/Training Failed for you?

**Comment by martinlarosa** [comment]
I got fixed late last night. Lets say 22hs ago

**Comment by jwiley84** [comment]
Good to know the training is working. Are you still having an issue with dispatch refresh?

**Comment by martinlarosa** [comment]
dispatch is working as the error was the same as in the portal I guess.
Regarding the original error of staging endpoint behaving as the production one, I'm not sure now as we published to both environments just to avoid the problem. But I believe that we solved it at some point by publishing to production, then Staging was working correctly but can't say at the moment if it is because it fixed something or if staging was still showing production jsut that this time they both actually had the same version.
